File: GGAndroid_2.0/GGPay/pay-sample/src/main/java/io/ganguo/pay/sample/PaySampleActivity.java
package io.ganguo.pay.sample;

import io.ganguo.pay.sample.viewmodel.PaySampleViewModel;
import io.ganguo.vmodel.ViewModelActivity;
import io.ganguo.viewmodel.databinding.IncludeHfRecyclerBinding;

/**
 * <p>
 * 支付相关Demo
 * </p>
 * Created by leo on 2018/8/7.
 */
public class PaySampleActivity extends ViewModelActivity<IncludeHfRecyclerBinding, PaySampleViewModel> {
    @Override
    public PaySampleViewModel createViewModel() {
        return new PaySampleViewModel();
    }

    @Override
    public void onViewAttached(PaySampleViewModel viewModel) {

    }
}

File: GGAndroid_3.0/main/app-pay-sample/src/main/java/io/ganguo/pay/sample/PaySampleActivity.kt
package io.ganguo.pay.sample

import io.ganguo.pay.sample.viewmodel.PaySampleViewModel
import io.ganguo.viewmodel.core.BaseViewModel
import io.ganguo.viewmodel.databinding.IncludeHfRecyclerBinding
import io.ganguo.viewmodel.core.view.ViewModelActivity

/**
 *
 *
 * 支付相关Demo
 *
 * Created by leo on 2018/8/7.
 */
class PaySampleActivity : ViewModelActivity<IncludeHfRecyclerBinding, PaySampleViewModel>() {
    override fun createViewModel(): PaySampleViewModel {
        return PaySampleViewModel()
    }

    override fun onViewAttached(viewModel: BaseViewModel<Any>) {

    }
}
